randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1971,17 +1971,23 @@ Extra - room_id: [42]
   > Event - Serris
       All of the following:
           Any of the following:
-              # Serris Weapon Requirements
-              Charge Beam or Missiles ≥ 10
-              Missiles ≥ 5 and 20 DMG Missiles
+              # Serris Weapon Requirements (50HP)
+              Charge Beam or Missiles ≥ 20
```

Comment:
this missile should have 10 dmg missile just in case

Reply:
20x5DMG missiles is 100DMG

Reply:
because of the lack of dmg specifier i didnt even know that you implicitly referenced 5dmg here, especially because we use this nowhere else in the logic db. I don't like that. Please change this in some way to make this either more obvious, or hard require 10 dmg missiles

Reply:
We seem to use it in a few places?
![image](https://github.com/user-attachments/assets/6195cbc2-cf3e-4358-ac72-96892edfe0b8)
![image](https://github.com/user-attachments/assets/7f0696a2-30da-4026-8467-bdd257bd03d5)

Reply:
and it doesn't even need to specifically be 5dmg, it's just the minimum possible. If you have any other missile upgrades it would and X number of missiles it also satisfies the requirement. just 5DMG is what needs to be considered for the calculation

Reply:
It just feels very inconsistent with how we've been doing all the other boss/combat logic (outside of templates)

Reply:
so what.. create a template for 5DMG missile and just make it =diffusion?? it seems so extra an unnecessary

Reply:
or would you rather just say no boss is expected to be killed with 5DMG/diffusion missiles and only ever use 10DMG and above templates for them?

Reply:
I prefer seeing an explicit "5 DMG Missiles" over "okay, how much damage did diffusion-only do again?"

We currently have no plans to have the possibility to use anything lower than 10 dmg missiles. Even with the future "any missile upgrade gives ability to fire missiles" option. Hence why basically no other combat logic bothers with it, and my suggestion of "we can just hard require 10 dmg missiles here"
And if we do change the plans, then we need to change quite a few places in the DB anyway, one more to the pile doesn't hurt.

Reply:
So yes, id prefer for now if we only use 10dmg+ for bosses.

Reply:
> I prefer seeing an explicit "5 DMG Missiles" over "okay, how much damage did diffusion-only do again?"
you don't need to do this though at all, you just need to say "hey I need 20 missiles regardless of dmg" which is pretty simple to understand...

I'm done arguing I'll just concede.